sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldContainAssertion.Xunit.Legacy.cs

```diff
@@ -0,0 +1,215 @@
+﻿namespace TestXunitLegacy
+{
+    using System;
+    using FluentAssertions;  // v4.9
+    using Xunit;
+    using Xunit.Extensions; // v1.9.1
```

Comment:
This is a copy of the existing non-legacy Xunit tests, with the _using_ statements fixed up to work with the legacy version.